web_action_conditionable/static/src/js/field_one2many.js

```diff
@@ -0,0 +1,38 @@
+/* global py */
+/* Copyright 2019 Alexandre Díaz
+ * License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). */
+odoo.define('web.web_action_conditionable', function (require) {
+    "use strict";
+
+    var FieldOne2Many = require('web.relational_fields').FieldOne2Many;
+
+    FieldOne2Many.include({
+        init: function () {
+            var self = this;
+            try {
+                return this._super.apply(this, arguments);
+            } catch (error) {
```

Comment:
I don't understand this structure. Are you only adding your stuff when upstream code fails? Why not adapt those attributes to booleans and avoid `_super()` to fail? :thinking:

Reply:
This follows the original implementation. This allows to have the "normal" behavior and the "plus" added by this module.